### PR TITLE
RFC 1: Governable Parameter Index!

### PR DESCRIPTION
**Pull request by beaushinkle**
Simple formatting PR - we take all of the existing governable parameters and collect them into an index.

tagging @michalinacienciala for review (feel free to merge if it looks fine to you).

### CODE REVIEW

File: docs/rfc/rfc-1.adoc [hunk 1]
```diff
@@ -445,3 +445,45 @@ unlucky operator should submit a transaction on-chain to multiply the fee by a
...
+  be included in a sweep.
+- `failed_heartbeat_reward_removal_period`: The amount of time an operator is
+  removed from reward eligibility after failing a heartbeat.
+- `group_size`: The total size of the signing group.
+- `heartbeat`: The number of group members required for a
+  <<heartbeat,heartbeat>> to be successful.
+- `heartbeat_block_length`: The number of ethereum blocks until the next heartbeat.
+  If set to 40, then the signers sign every 40th block.
+- `sweep_max_btc`: The amount of summed unswept bitcoin deposits that will
+  trigger an early sweep.
```
Comment (michalinacienciala) [inline docs/rfc/rfc-1.adoc:467]: We can consider this change to make the definition more precise: 
```suggestion
- `sweep_max_btc`: The amount of summed non-dust unswept bitcoin deposits that
  will trigger an early sweep on a wallet.
```

File: docs/rfc/rfc-1.adoc [hunk 2]
```diff
@@ -445,3 +445,45 @@ unlucky operator should submit a transaction on-chain to multiply the fee by a
...
+- `sweep_period`: The amount of time we wait between sweeps.
```
Comment (michalinacienciala) [inline docs/rfc/rfc-1.adoc:468]: We can consider this change to make the definition more precise: 
```suggestion
- `sweep_period`: The amount of time we wait between scheduled sweeps on a wallet.
```

File: docs/rfc/rfc-1.adoc [hunk 3]
```diff
@@ -445,3 +445,45 @@ unlucky operator should submit a transaction on-chain to multiply the fee by a
...
+== Governable Parameters
+Alphabetized list of Governable Parameters with additional notes.
+
+
+- `bridge_deposit_flat_fee`: The flat fee for depositing bitcoin.
+- `bridge_deposit_percentage_fee`: The percentage fee for depositing bitcoin.
+- `dkg`: The minimum number of members we're allowed to drop down to during the
+  DKG group formation re-try period.
+- `dust_threshold`: The minimum bitcoin deposit amount for the transaction to
+  be included in a sweep.
```
Comment (michalinacienciala) [inline docs/rfc/rfc-1.adoc:458]: We can consider this change to make the definition more precise: 
```suggestion
- `dust_threshold`: The minimum bitcoin deposit amount for the transaction to
  be considered for a sweep.
```

**Review by michalinacienciala — COMMENTED** [review]
Left couple of small comments

File: docs/rfc/rfc-1.adoc (hunk 1 above)
Comment (beaushinkle) [reply docs/rfc/rfc-1.adoc]: resolved in f980fa7

File: docs/rfc/rfc-1.adoc (hunk 2 above)
Comment (beaushinkle) [reply docs/rfc/rfc-1.adoc]: resolved in 3829379

File: docs/rfc/rfc-1.adoc (hunk 3 above)
Comment (beaushinkle) [reply docs/rfc/rfc-1.adoc]: resolved in e14bfdd

**Review by michalinacienciala — APPROVED** [review]

**Pull request merged**